recipe-server/normandy/settings.py

```diff
@@ -328,6 +329,11 @@ def RAVEN_CONFIG(self):
 
     PROD_DETAILS_DIR = values.Value(os.path.join(Core.BASE_DIR, 'product_details'))
 
+    # AWS settings
+    AWS_ACCESS_KEY_ID = values.Value()
+    AWS_SECRET_ACCESS_KEY = values.Value()
+    AWS_STORAGE_BUCKET_NAME = values.Value()
```

Comment:
to clarify here, I will only be setting `DJANGO_AWS_STORAGE_BUCKET_NAME`, because not setting the other two should cause the backend to fall back to boto3's default ways of determining credentials, in my case ec2 instance profile credentials. You can verify that this will work by supplying aws credentials as `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` (without the `DJANGO_` prefix), because env vars is one of the fallbacks for boto3.